Topic5Mysql/src/DataBaseTest.java

```diff
@@ -0,0 +1,68 @@
+import java.net.ConnectException;
+import java.sql.ResultSet;
+
+public class DataBaseTest {
+
+	public static void main(String[] args) throws Exception {
+
+		// Connection
+
+		DataBaseConnection dataBaseConnection = new DataBaseConnection();
+		dataBaseConnection.connect();
+
+		// Test exercise 4
+
+		System.out.println(" ");
+		System.out.println(" ---Excercise 4--- ");
+		System.out.println(" ");
+
+		DataBaseManager dataBaseManager = new DataBaseManager();
+
+		ResultSet resultSet1;
+		ResultSet resultSet2;
+
+		int course_id = 2;
+
+		String query1 = "SELECT name, first_name, last_name FROM (teacher T NATURAL JOIN course C) WHERE (C.course_id=" + course_id

```

Comment:
Looks good, but why did you use NATURAL JOIN?
It's a good practice to analyze in each case which kind of JOIN you need to use: In this case I would use an INNER JOIN since we need to retrieve all the records in the left table (Course) that have a matching record in the right table (Teacher), but if we are looking for all the courses independetly if they have a teacher assigned or not we should use LEFT JOIN.
In the following page you have an excellent Visual Representation of SQL JOINS -> http://www.codeproject.com/Articles/33052/Visual-Representation-of-SQL-Joins